rabbitmq/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM rabbitmq:3.11-management
```

Comment:
Please add this file in the manual_maintenance template under 
### Dockerfile updates

Reply:
done

Reply:
pl